languages/python/sqlalchemy-oso/tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py3-sqlalchemy{13,14}
+
+[testenv]
+usedevelop = true
+deps =
+    -rrequirements-test.txt
+    sqlalchemy13: SQLAlchemy==1.3.0
```

Comment:
We should test on the latest versions of SQLAlchemy 1.3 and 1.4 so that we know that we support new versions properly when they are released.

Reply:
I figured we didn't have to since SQLAlchemy follows semver, so if `==1.3.0` passes then anything in the `~=1.3.1` range should pass (and same for `==1.4.0` and `~=1.4.1`).

Reply:
Offline: Test on .0 and latest